.github/workflows/maven.yaml

```diff
@@ -2,6 +2,10 @@ name: Publish icu4j.jar/utilities.jar to GH Maven
 on:
   release:
     types: [created]
+
+permissions:
+  contents: read
```

Comment:
this workflow needs to `mvn deploy` to github packages

Reply:
in fact that's the only purpose of this workflow

Reply:
so i'd just revert these two lines.

Reply:
Sorry! Fixed in https://github.com/unicode-org/icu/pull/2529.